hercules/plant_components/electrolyzer_plant.py

```diff
@@ -115,7 +210,7 @@ def step(self, h_dict):
                 - power_input_kw: Power input to electrolyzer [kW]
         """
         # Gather inputs
-        local_power = h_dict["locally_generated_power"]  # TODO check what units this is in
+        local_power = h_dict["plant"]["locally_generated_power"]  # TODO check what units this is in
```

Comment:
`locally_generated_power` is in kW, so we could remove the TODO here and explain that the units are kW (or remove the comment entirely, since Hercules is now uniformly in units of kW)